std/range/package.d

```diff
@@ -4696,7 +4702,6 @@ body
 
```

Comment:
I think probably you can remove the import here, if not using unsigned in here any more.